test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/statuses/CaseStatusViewSpec.scala

```diff
@@ -27,11 +27,24 @@ class CaseStatusViewSpec extends ViewSpec {
 
   "Case Status" should {
 
+    "render CANCELLED without the reason code" in {
```

Comment:
Is this just to test the logic?  Cancelled should always have a reason.

Reply:
Yeh but from a data model perspective it isn't essential, and we still want it to show in Grey

Reply:
It just saves Testers any headaches from changing the status in the DB and wondering why it still shows in blue